### ISSUE TITLE
spec copyedits

### ISSUE THREAD

**Issue by chalin** [dart-lang/sdk#23781]
BNF “syntax errors” found during [automated grammar extraction](https://github.com/chalin/dart-spec-and-grammar). Prose copyedits found during review.

https://codereview.chromium.org/1217533003

**Comment by eernstg** [comment]
No milestone now: This does not block Dart 2.

Note that a larger update of grammar related parts of the specification is planned, based on [Dart.g](https://github.com/dart-lang/sdk/blob/master/docs/language/Dart.g). This issue will be consulted as part of this work.

**Comment by eernstg** [comment]
Performed the copyedits which were still needed (many locations had already been fixed, or the text is very different today) in [this CL](https://dart-review.googlesource.com/90223).

**Comment by chalin** [comment]
> Better late than never ;-)

Yes! I'm glad that the updates are being done, thanks. cc @kwalrath @sfshaza2.
